astropy/utils/data.py

```diff
@@ -968,6 +968,11 @@ def download_file(remote_url, cache=False, show_progress=True, timeout=None):
         The timeout, in seconds.  Otherwise, use
         `astropy.utils.data.Conf.remote_timeout`.
 
+    pkgname : `str`, optional
+        The package name to use to locate the download cache. i.e. for
+        ``pkgname='astropy'`` the default cache location is
+        ``~/.astropy/cache``.
```

Comment:
I am not entirely sure what the behaviour is here, so I think this is probably the best way to doc it.

Reply:
Maybe instead of assuming it is `~/.astropy`, crosslink to the config doc, since over there, it explains that user can further customize with XDG stuff?

Reply:
To be honest I am going to open another PR after this which will change all the directories stuff anyway, so I don't really think it's worth going down the pit with.